less/catppuccin-accents.less

```diff
@@ -0,0 +1,45 @@
+s#accent(@accent) when (@accent =rosewater) {
```

Comment:
`s` at the start of the line should be removed. c:

Reply:
Oh, also.. just from a typographic point I'd highly prefer a space after the `=` as well.